integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -574,11 +574,34 @@ export class CdcJsService implements OnDestroy {
           if (currentEmail !== userDetails.uid) {
             this.logoutUser();
           }
+          this.handleProfileUpdateToastMessage(response);
         });
       });
     }
   }
 
+  /**
+   * handle toast message on profile update
+   * @param response
+   *
+   */
+  handleProfileUpdateToastMessage(response?: any) {
```

Comment:
MEDIUM: can we make this method `protected` as it is used only within this class?